src/platform-includes/getting-started-install/javascript.angular.mdx

```diff
@@ -1,18 +1,32 @@
 ```bash {tabTitle:npm}
+# Angular 12 and newer:
+npm install --save @sentry/angular-ivy @sentry/tracing
+
+# Angular 10 and 11:
 npm install --save @sentry/angular @sentry/tracing
 ```
 
 ```bash {tabTitle:Yarn}
+# Angular 12 and newer:
+yarn add @sentry/angular-ivy @sentry/tracing
+
+# Angular 10 and 11:
 yarn add @sentry/angular @sentry/tracing
 ```
 
-<Alert level="info" title="Angular Version Compatibility" >
+### Angular Version Compatibility
+
+Because of how Angular libraries are compiled, you need to use the correct version of the Sentry SDK for your Angular version:
```

Comment:
```suggestion
Because of the way Angular libraries are compiled, you need to use a specific version of the Sentry SDK for each corresponding version of Angular as shown below:
```